tests/deployment/switch_config.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2013-2014 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied.  See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Deployment Tests - These tests are intended for our
+internal setup only and will most likely not work on
+other HaaS configurations. These tests are for dell and cisco switches"""
+
+from haas import api, model, deferred, server
+from haas.model import db
+from haas.test_common import *
+import pytest
+
+
+@pytest.fixture
+def configure():
+    config_testsuite()
+    config_merge({
+        'dell': {
+            'save': 'True'
+         }
+    })
+    config.load_extensions()
+
+
+fail_on_log_warnings = pytest.fixture(autouse=True)(fail_on_log_warnings)
+fresh_database = pytest.fixture(fresh_database)
+
+
+@pytest.fixture
+def server_init():
+    server.register_drivers()
+    server.validate_state()
+
+
+with_request_context = pytest.yield_fixture(with_request_context)
+
+site_layout = pytest.fixture(site_layout)
+
+pytestmark = pytest.mark.usefixtures('configure',
+                                     'server_init',
+                                     'fresh_database',
+                                     'with_request_context',
+                                     'site_layout')
+
+
+class TestNativeNetwork(NetworkTest):
```

Comment:
Would you change this to a more descriptive name, add add a docstring explaining what this test is for?

Also, this file is dell-specific; if that's the case I'd like it to be called out more prominently, and the file probably out to be separated from the rest of the deployment tests so you can still do py.test tests/deployment if you don't have a dell switch in your setup. The other option is to use `pytest.skip()`